### DISCUSSION TITLE
Có thể bấm Backspace lại từ sai để sửa nó không?

### DISCUSSION THREAD

**Discussion by manhtq0508** [BambooEngine/ibus-bamboo#510] [Q&A]
Mình có cài Ibus-Bamboo trên Arch thấy hoạt động rất tốt và ổn định như gõ trên Windows
Có 1 điều khác là bên Windows là khi mình gõ sai 1 từ, mình có thể bấm Backspace để quay lại từ sai đó để sửa mà không phải gõ lại từ đầu. Nhưng Ibus-Bamboo lại không như vậy, khi gõ sai thì buộc phải gõ lại cả từ chứ không thể sửa được. 
Mình muốn hỏi là đây là hạn chế của Linux hay do mình cấu hình chưa đúng? (Mình cài thông qua hướng dẫn ở tệp README.md cho Arch và không làm gì thêm)
Mình mới qua nên cũng chưa rõ về Linux lắm. Cảm ơn mọi người đã đọc.

**Comment by goatastronaut0212** [comment]
@manhtq0508 Mình không rõ là Ibus-bamboo có tính năng nào như vậy lắm nhưng với mình thì nó không quá khó chịu. Nhưng nếu như phải có tính năng trên như bạn nói thì mình nghĩ có thể nếu hỗ trợ tính năng tiên đoán từ có thể sẽ hay hơn. Hiện tại thì vẫn chưa có ai làm nó cả. https://github.com/BambooEngine/ibus-bamboo/issues/368.

Đây không phải là hạn chế của Linux. Ibus-bamboo là phần mềm mã nguồn mở chỉ là nó hiện không có tính năng bạn đề cập tới. Nếu bạn cần tính năng nào đó và muốn đề xuất, viết code hoặc theo dõi đừng ngần ngại tạo 1 issue hay pull request mới.
